src/omotes_simulator_core/entities/network_controller.py

```diff
@@ -44,6 +44,23 @@ def __init__(
         self.producers = producers
         self.consumers = consumers
         self.storages = storages
+        self._set_priority_from_marginal_costs()
+
+    def _set_priority_from_marginal_costs(self) -> None:
+        """This method sets the priority of the producers based on the marginal costs."""
```

Comment:
Can you add a small explanation in the docstring about the method?

Reply:
done